src/cmd/service.go

```diff
@@ -214,6 +216,17 @@ func init() {
 	importCmd.AddCommand(importServicesCmd)
 }
 
+func hydrateService(service *opslevel.Service, client *opslevel.Client) {
+	_, err := service.GetDependencies(client, nil)
+	cobra.CheckErr(err)
+	_, err = service.GetDependents(client, nil)
+	cobra.CheckErr(err)
+	_, err = service.GetDependencies(client, nil)
```

Comment:
Function has been called twice

Reply:
🤦‍♂️ we just realllly need those dependencies 😬